test/test_accumulator.jl

```diff
@@ -43,14 +43,14 @@ push!(ct, ct2)
 @test ct["c"] == 2
 
 ct3 = counter(Dict([("a",10), ("b",20)]))
-@test isa(ct3, Accumulator{ASCIIString,Int})
```

Comment:
I can do that, but is there a good reason we should prolong the use of `ASCIIString`?